### ISSUE TITLE
Can't verify Linux Mint image using address or local files

### ISSUE THREAD

**Issue by JoeJoeTV** [linuxmint/mintstick#115]
- Specs:
```
OS: Linux Mint Cinnamon 21.1
```

- Problem Description:
I tried verifying a Linux Mint 21.1 ISO file, but everytime I try, I simply get an error screen in the application with the error "name 'sums_path' is not defined". THis happens when trying to use the files from the URL or with local files.
From searching through the source I found [this line](https://github.com/linuxmint/mintstick/blob/da256a662f363136fb6a60b9407abc434b7b4d24/lib/verify.py#L280), which seems to be the only line, where the variable "sums_path" is used at all. I only did a quick search, so this could be irrelevant.

**Comment by mtwebster** [comment]
Hi, could I get you to try out a fix for this?

The package is here:
https://github.com/linuxmint/mintstick/releases/tag/master.mint21

Download the 'packages.tar.gz' file, unpack it and install the .deb file in it (you should be able to just double-click it from the file manager, or open a terminal where you extracted it and run `sudo dpkg -i *.deb`).

Or, if you don't mind editing files, open `/usr/lib/mintstick/verify.py` and change that `sums_path` to `PATH_SUMS`.

Thanks!

**Comment by JoeJoeTV** [comment]
Ok, I tried the fix by just editing the python file. Now the error mentioned earlier doesn't apprear anymore, but it tells me that the key wasn't found on the keyserver. I checked the sha256sums file and the sum of the mint 21.1 iso seems to match.
Also the "Verify" button only gets unlocked by switching the tabs back and forth one time.

**Comment by mtwebster** [comment]
Strange - works ok here.  When you first start it, the verify button can take a few moments to become clickable. It should probably show that it's busy somehow.

Can you try running it from a terminal?
```
mintstick -m iso --iso *path-to-iso*
```

**Comment by JoeJoeTV** [comment]
I know that the button is greyed out when generating the hash, but I mean after the hash is generated and displayed and  I change the tab to local files, even after selecting the files, it's greyed out. If I then switch to url and back again, I can click it.
With the command you provided it shows the same error.
I also ran it with --debug and this is the output:
```Log
Checking  /media/joejoetv/Ventoy/linuxmint-21.1-cinnamon-64bit.iso
Importing 27DEB15644C6B3CF3BD7D291300F846BA25BAE09
Deleting 27DEB15644C6B3CF3BD7D291300F846BA25BAE09
```

**Comment by clefebvre** [comment]
Button sensitivity fixed in upcoming 1.6.0.
